src/content-helper/common/utils/post.tsx

```diff
@@ -65,7 +65,7 @@ export function PostListItemMetric(
 ): JSX.Element {
 	if ( 'views' === metric ) {
 		return (
-			<span className="parsely-top-post-metric-data">
+			<span className="parsely-post-metric-data">
```

Comment:
Updated this name to be more neutral since this code is shared between the Dashboard Widget and Sidebar Related Posts.